target/linux/ramips/base-files/etc/board.d/01_leds

```diff
@@ -413,6 +413,10 @@ youku-yk1)
 	set_wifi_led "$board:blue:air"
 	set_usb_led "$board:blue:usb"
 	;;
+atel-u270)
+	ucidef_set_led_default "power" "power" "$board:orange:power" "0"
```

Comment:
Drop this line here and use it diag.sh instead to indicate the boot status. In case the red power led is switched on by the bootloader it can be disabled in the dts using ```default-state = "off";```